packages/jv-ui-components/material-ui/Stack/Stack.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2024. Cloud Software Group, Inc.
+ * This file is subject to the license terms contained
+ * in the license file that is distributed with this file.
+ */
+
+import { forwardRef } from "react";
+import { Stack as MuiStack } from "@mui/material";
+import { StackProps } from "@mui/material/Stack/Stack";
+
+export const Stack = forwardRef((props: StackProps, ref) => {
```

Comment:
What value does this add over using MUI directly?

Reply:
While it might seem like just a simple wrapper around the MUI Stack right now, it actually unlocks some really useful features for us.  For example, Anna can now easily add custom styles using the `jv-` prefix, which keeps our styling consistent and organized. Plus, wrapping MUI components is an important guideline for our project, ensuring we maintain a standardized approach throughout.

Reply:
Cool, thanks!